<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BibleMemorizer</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="mMainSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QFrame" name="mNavFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="mRemoveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>360</y>
          <width>194</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove &amp;Verse...</string>
        </property>
        <property name="shortcut">
         <string>Alt+V</string>
        </property>
       </widget>
       <widget class="QLabel" name="openForLeadLabel">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>217</y>
          <width>63</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Open For:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="mOpenButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>284</y>
          <width>194</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mAddButton">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>67</y>
          <width>194</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Add New Verse</string>
        </property>
        <property name="shortcut">
         <string>Alt+A</string>
        </property>
       </widget>
       <widget class="QComboBox" name="mOpenForComboBox">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>143</y>
          <width>194</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QTabWidget" name="mTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="welcome">
        <attribute name="title">
         <string>&amp;Categories</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="mCategoryNoticeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>You can use the navigation on the left to select a verse to open.  Here you can edit the categories used to sort verses.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="mCategoryListBox">
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mAddCategoryButton">
           <property name="text">
            <string>A&amp;dd Category...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mRenameCategoryButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Re&amp;name Category...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mRemoveCategoryButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove Cate&amp;gory...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileNewAction"/>
    <addaction name="fileOpenAction"/>
    <addaction name="fileSaveAction"/>
    <addaction name="fileSaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="fileExitAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="editUndoAction"/>
    <addaction name="editRedoAction"/>
    <addaction name="separator"/>
    <addaction name="editCutAction"/>
    <addaction name="editCopyAction"/>
    <addaction name="editPasteAction"/>
   </widget>
   <widget class="QMenu" name="Quiz">
    <property name="title">
     <string>&amp;Quiz</string>
    </property>
    <addaction name="quizTextAction"/>
    <addaction name="quizReferenceAction"/>
   </widget>
   <widget class="QMenu" name="preferencesMenu">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
    <addaction name="preferencesSettingsAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="Quiz"/>
   <addaction name="preferencesMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="fileNewAction">
   <property name="icon">
    <iconset>
     <normaloff>image0</normaloff>image0</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset>
     <normaloff>image1</normaloff>image1</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="icon">
    <iconset>
     <normaloff>image2</normaloff>image2</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="filePrintAction">
   <property name="icon">
    <iconset>
     <normaloff>image3</normaloff>image3</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="iconText">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="fileExitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="editUndoAction">
   <property name="icon">
    <iconset>
     <normaloff>image4</normaloff>image4</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="iconText">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="editRedoAction">
   <property name="icon">
    <iconset>
     <normaloff>image5</normaloff>image5</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="iconText">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="editCutAction">
   <property name="icon">
    <iconset>
     <normaloff>image6</normaloff>image6</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="iconText">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="editCopyAction">
   <property name="icon">
    <iconset>
     <normaloff>image7</normaloff>image7</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="iconText">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="editPasteAction">
   <property name="icon">
    <iconset>
     <normaloff>image8</normaloff>image8</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="iconText">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="helpAboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="preferencesSettingsAction">
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="iconText">
    <string>Settings...</string>
   </property>
  </action>
  <action name="quizTextAction">
   <property name="text">
    <string>Text Quiz</string>
   </property>
   <property name="iconText">
    <string>Text Quiz</string>
   </property>
  </action>
  <action name="quizReferenceAction">
   <property name="text">
    <string>Reference Quiz</string>
   </property>
   <property name="iconText">
    <string>Reference Quiz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mTabWidget</tabstop>
  <tabstop>mCategoryListBox</tabstop>
  <tabstop>mAddCategoryButton</tabstop>
  <tabstop>mRenameCategoryButton</tabstop>
  <tabstop>mRemoveCategoryButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>filePrintAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>filePrint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editUndoAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>editUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editRedoAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>editRedo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>editCut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCopyAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>editCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editPasteAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>editPaste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileExitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>fileExitAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpAboutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>helpAboutAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileOpenAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>fileOpenAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>fileSaveAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSaveAsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>fileSaveAsAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileNewAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>fileNewAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAddCategoryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainUI</receiver>
   <slot>mAddCategoryButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRenameCategoryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainUI</receiver>
   <slot>mRenameCategoryButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRemoveCategoryButton</sender>
   <signal>clicked()</signal>
   <receiver>MainUI</receiver>
   <slot>mRemoveCategoryButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preferencesSettingsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>preferencesSettingsAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quizTextAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>quizTextAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quizReferenceAction</sender>
   <signal>triggered()</signal>
   <receiver>MainUI</receiver>
   <slot>quizReferenceAction_activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAddButton</sender>
   <signal>clicked()</signal>
   <receiver>MainUI</receiver>
   <slot>mAddButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOpenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainUI</receiver>
   <slot>mOpenButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRemoveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainUI</receiver>
   <slot>mRemoveButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileExitAction_activated()</slot>
  <slot>helpAboutAction_activated()</slot>
  <slot>fileNewAction_activated()</slot>
  <slot>fileSaveAsAction_activated()</slot>
  <slot>fileSaveAction_activated()</slot>
  <slot>fileOpenAction_activated()</slot>
  <slot>preferencesSettingsAction_activated()</slot>
  <slot>quizTextAction_activated()</slot>
  <slot>quizReferenceAction_activated()</slot>
 </slots>
</ui>
